indexer/altitude_loader.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "indexer/feature_altitude.hpp"
+#include "indexer/index.hpp"
+
+#include "coding/memory_region.hpp"
+
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+#include "3party/succinct/rs_bit_vector.hpp"
+
+namespace feature
+{
+class AltitudeLoader
+{
+public:
+  explicit AltitudeLoader(MwmValue const & mwmValue);
+
+  TAltitudes const & GetAltitudes(uint32_t featureId, size_t pointCount) const;
```

Comment:
Up to you. From my point of view, mutable may hide important signal that a const method actually changes something in the class state, therefore it may be not safe to use this class in the multithreaded environment. On the other hand, if used properly, it may be helpful. So, the final decision is yours.